spec.txt

```diff
@@ -7493,10 +7485,11 @@ Although [link titles] may span multiple lines, they may not contain
 a [blank line].
 
 An [inline link](@) consists of a [link text] followed immediately
-by a left parenthesis `(`, optional [whitespace], an optional
-[link destination], an optional [link title] separated from the link
-destination by [whitespace], optional [whitespace], and a right
-parenthesis `)`. The link's text consists of the inlines contained
+by a left parenthesis `(`, optional spaces, tabs, and up to one line ending,
+an optional [link destination], an optional [link title] separated from the link
+destination by spaces, tabs, and up to one line ending, optional spaces, tabs,
+and up to one line ending, and a right parenthesis `)`.
+The link's text consists of the inlines contained
```

Comment:
Maybe use a bullet list here to avoid confusion?  At first I thought "an optional [link title] separated from the link  destination by spaces, tabs, and up to one line ending, optional spaces, tabs,  and up to one line ending," would allow up to two line endings...but they belong to separate clauses.